packages/perseus-core/src/parse-perseus-json/perseus-parsers/widgets-map.test.ts

```diff
@@ -37,9 +37,10 @@ describe("parseWidgetsMap", () => {
         );
     });
 
-    it("rejects a widget ID numbered 0", () => {
-        // Widget IDs with 0 currently cause a full-page crash when the
-        // exercise is rendered in webapp!
+    it("accepts a widget ID numbered 0", () => {
+        // Widget IDs with 0 cause a full-page crash when an exercise is
```

Comment:
Do we need a ticket to address this?

Reply:
Eh, probably not? We're not writing content with IDs of 0 into datastore anymore. And Jeremy has talked about wanting the IDs to be completely opaque strings, so addressing the issue with 0 separately seems like maybe a waste of effort.